test/dlc_tests/sanity/quick_checks/test_yml_config.py

```diff
@@ -0,0 +1,57 @@
+import os
+import re
+
+import pytest
+import yaml
+
+from test.test_utils import is_pr_context
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_training.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_training_yml():
+    _release_images_yml_verifier(image_type="training", excluded_image_type="inference")
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_inference.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_inference_yml():
+    _release_images_yml_verifier(image_type="inference", excluded_image_type="training")
+
+
+def _release_images_yml_verifier(image_type, excluded_image_type):
```

Comment:
Do you we see a case where we may have more than 2 image types in the future? If yes, then we should make excluded image type parameter a list of matching patterns.
Alternatively, make _release_images_yml_verifier one parameter-function with image_type as parameter; then convert yml into dictionary and for each entry search and validate that each entry matches the key structure of [framework, version, customer_type, arch_type, image_type].

Reply:
+1 to @kenny-ezirim's comment.

In fact, would it be better to make the parameter "allowed_image_types", so that even if we do have multiple image types, we can release "combinations" of image types from these files.

Or is this antithetical to the idea of having separate yml files?

Reply:
We will update test optimizations in subsequent PR to prioritize getting this functionality available to users